### PR TITLE
Add labels to minutes

### LINKED ISSUE #86: Show minute label on minutes
Currently, the labels are not shown. See 1327.
image

### PR DESCRIPTION
**Pull request by frcroth**
Closes #86

### CODE REVIEW

**Review by jeriox — APPROVED** [review]

File: myhpi/core/templates/core/minutes.html [hunk 1]
```diff
@@ -25,6 +25,11 @@ <h4>Participants</h4>
                     <li>{{ participant.get_full_name }}</li>
                 {% endfor %}
             </ul>
+            {%  if page.labels.all %}
```
Comment (jeriox) [inline myhpi/core/templates/core/minutes.html:28]: maybe we want to add an heading here? 1327 reads "Labels" above the actual labels

Comment (jeriox) [reply myhpi/core/templates/core/minutes.html]: additional wish: only render the headline if there are labels

Comment (frcroth) [reply myhpi/core/templates/core/minutes.html]: 1327 does something different:
image

What is it?

Comment (frcroth) [reply myhpi/core/templates/core/minutes.html]: Then approve please

Comment (jeriox) [reply myhpi/core/templates/core/minutes.html]: i already did at 11:16

**Pull request merged**